ledger/accountdb.go

```diff
@@ -3940,16 +3954,26 @@ func (mc *MerkleCommitter) LoadPage(page uint64) (content []byte, err error) {
 	return content, nil
 }
 
+// catchpointAccountResourceCounter keeps track of the resources processed for the current account
+type catchpointAccountResourceCounter struct {
+	totalAppParams      uint64
```

Comment:
im using these fields in other structs, not sure how to suppress this warning

Reply:
1. rename `catchpointAccountResourceCounter` to acctResCnt
2. use `iterator.acctResCnt.totalAppParams` instead of `iterator.totalAppParams`